resources/function_help/json/array_to_string

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "array_to_string",
+  "type": "function",
+  "description": "Concatenates array elements into a string separated by a delimiter using and optional string for empty values.",
+  "variableLenArguments": true,
+  "arguments": [
+    {"arg":"array", "description":"the input array"},
+    {"arg":"delimiter", "description":"the string delimiter used to separate concatenated array elements"},
+    {"arg":"empty_string", "description":"the optional string to use as replacement to empty values"}],
```

Comment:
Check out https://github.com/qgis/QGIS/blob/master/resources/function_help/json/single_sided_buffer for the correct way to handle optional arguments